File: Cplusplus/URI/01-iniciantes/02-Nivel/1005.cpp
#include <iostream>
#include <iomanip>
 
using namespace std;
 
int main()
{
    double firstGrade, secondGrade , averageGrade;

    
    cout << "Digite a primeira nota: " << endl;
    cin  >> firstGrade;
    cout << "Digite a segunda nota: " << endl;
    cin  >> secondGrade; 
    averageGrade = ((firstGrade * 3.5) + (secondGrade * 7.5))/ 11.0;
    cout << "MEDIA = " << fixed << setprecision(5) << averageGrade << endl;   
 
    return 0;
}

File: Javascript/URI/01-Iniciantes/1000-1020/1005.js
let firstGrade, secondGrade , averageGrade;

firstGrade = parseFloat(prompt("Digite a primeira nota"));

secondGrade =  parseFloat(prompt("Digite a segunda nota"));
averageGrade = ((firstGrade * 3.5) + (secondGrade * 7.5))/ 11.0;
console.log(`MEDIA = ${averageGrade.toFixed(5)}`);
alert(`MEDIA = ${averageGrade.toFixed(5)}`);